docs/topics/media-pipeline.rst

```diff
@@ -198,23 +198,25 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_VERIFY = False # or True (None by default)
 
 
-Custom Amazon S3 headers in FileStorage
+Custom Amazon S3 headers
+'''''''''''''''''''''''''
 
-Steps to pass **SSEKMSKeyId** and **ServerSideEncryption** to **scrapy** FilesPipeline using **AWS S3**
+Steps to pass **Headers** to **scrapy** FilesPipeline using **AWS S3**
 
-Configuration Steps::
+Configuration Steps:
```

Comment:
:lipstick: What about merging these into a single paragraph?

Also, I think the ```**boldface**``` is not necessary here.

Reply:
Merged into a paragraph and boldface is removed